src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/MissingSwitchDefaultCheckTest.java

```diff
@@ -72,6 +72,24 @@ public void testMissingSwitchDefaultSwitchExpressions() throws Exception {
             expected);
     }
 
+    @Test
+    public void testInputMissingSwitchDefaultCheckSwitchExpressionsFour() throws Exception {
+        final String[] expected = CommonUtil.EMPTY_STRING_ARRAY;
+        verifyWithInlineConfigParser(
+                getNonCompilablePath("InputMissingSwitchDefaultCheckSwitchExpressionsFour.java"),
+            expected);
+    }
+
+    @Test
+    public void testInputMissingSwitchDefaultCheckSwitchExpressionsFive() throws Exception {
```

Comment:
Similar to above, if we can't meaningfully name tests, we need to reconsider what we are testing.

Reply:
Tried to add @nrmancuso Please check